pytorch_lightning/loggers/wandb.py

```diff
@@ -217,6 +374,53 @@ def log_metrics(self, metrics: Dict[str, float], step: Optional[int] = None) ->
         else:
             self.experiment.log(metrics)
 
+    @rank_zero_only
+    def log_table(
+        self,
+        key: str,
+        columns: List[str] = None,
+        data: List[List[Any]] = None,
+        dataframe: "pandas.DataFrame" = None,
+        step: Optional[int] = None,
+    ) -> None:
+        """Log a Table containing any object type (text, image, audio, video, molecule, html, etc).
+
+        Can be defined either with `columns` and `data` or with `dataframe`.
+        """
+
+        metrics = {key: wandb.Table(columns=columns, data=data, dataframe=dataframe)}
+        self.log_metrics(metrics, step)
+
+    @rank_zero_only
+    def log_text(
+        self,
+        key: str,
+        columns: List[str] = None,
+        data: List[List[str]] = None,
+        dataframe: "pandas.DataFrame" = None,
+        step: Optional[int] = None,
+    ) -> None:
+        """Log text as a Table.
+
+        Can be defined either with `columns` and `data` or with `dataframe`.
+        """
+
+        self.log_table(key, columns, data, dataframe, step)
+
+    @rank_zero_only
+    def log_images(self, key: str, images: List[Any], **kwargs: str) -> None:
```

Comment:
For consistency with other loggers, `log_image` should be used there.

Reply:
The issue here is I wanted to make it explicit that it's a list of images (vs a single image).
In W&B, images need to be logged in batch to have a meaningful representation in the UI (and for better efficiency).
We just need to make sure that users won't loop through their images calling this method on each individual sample.
What do you think?

Reply:
@borisdayma that sounds reasonable. IMO it could be named log_image (singular) and still log either a single image or a list/batch of them.

Reply:
yeah, need to give it more thoughts. I want at least to force users to pass list of images so they don't try to loop over images which would give a sub-optimal experience in the UI. Calling it `log_images` (and not supporting `log_image`) would make it obvious.

Reply:
I changed it to `log_image`